packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -72,23 +78,18 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
 
   newId = actions.editor.internal.ensureSandboxId(newId);
 
-  if (state.live.isLive) {
-    actions.live.internal.disconnect();
+  // This happens when we fork. This can be avoided with state first routing
+  if (state.editor.isForkingSandbox) {
+    return;
   }
 
-  if (state.editor.sandboxes[newId] && !state.editor.sandboxes[newId].team) {
-    const sandbox = await effects.api.getSandbox(newId);
```

Comment:
When opening a team sandbox we automatically open a live session, we need to make sure this still happens. I'll test it later.

Reply:
Wow this works really well, nice work!